File: src/commands/misc/ping.js
module.exports = {
    name: 'ping',
    description: 'Pong!',
    needsAdmin: false,

    callback: (client, interaction) => {
        interaction.reply(`Pong!`);
        
        setTimeout(() => {
            interaction.deleteReply()
        }, ((5 * 60) * 1000));
    }
}

File: commands/ping.py
import discord
from discord.ext import commands

intents = discord.Intents.default()
intents.message_content = True

client = commands.Bot(command_prefix='/', intents=intents)


@commands.hybrid_command(name="ping", description="Ping!")
async def ping(ctx):
    await ctx.send(f'Pong!')


async def setup(client):
    client.add_command(ping)

